app/models/site_config.rb

```diff
@@ -140,6 +140,8 @@ class SiteConfig < RailsSettings::Base
   field :rate_limit_follow_count_daily, type: :integer, default: 500
   field :rate_limit_comment_creation, type: :integer, default: 9
   field :rate_limit_comment_antispam_creation, type: :integer, default: 1
+  # Explicitly defaults to 7 to accommodate DEV Top 7 Posts
+  field :rate_limit_mention_creation, type: :integer, default: 7
```

Comment:
@citizen428, maybe you know the answer to this: do I actually need to add this here? I added it because I followed the pattern that was set for the other values in `Settings::RateLimit`, but since this is a _new_ value, maybe it's not necessary? 🤔

Reply:
No, you don't need it here. The reason the old ones are still in `SiteConfig` is that we needed them for the data update script, eventually, these will all be removed, please don't add anything new here.

Reply:
Sounds good; I went ahead and [removed it](https://github.com/forem/forem/pull/13736/commits/7900642997ee2bb64343903b382ce9054a4f154f). Thanks for the gut check! ✅